README.md

```diff
@@ -27,15 +27,34 @@ run the following.
 $ juju-verify shutdown --units nova-compute/0 nova-compute/1
 ```
 
-**NOTE:** If you run check on multiple units at the same time, they must all run
-the same charm. Trying, for example, `juju-verify shutdown --units nova-compute/1
-ceph-osd/0` will result in error.
+## Known limitations
 
-Alternatively, a machine can be targeted:
+1. If you run check on multiple units at the same time, they must all run
+   the same charm. Trying, for example, `juju-verify shutdown --units nova-compute/1
+   ceph-osd/0` will result in error.
 
-```bash
-$ juju-verify shutdown --machines 0
-```
+   Alternatively, a machine can be targeted:
+
+   ```bash
+   $ juju-verify shutdown --machines 0
+   ```
+
+2. If you run a check on unit, which contains a subordinate unit, you will got only
+   a warning message about existence of the subordinate unit. To check if it is safe
+   to remove this unit, you must perform a check on this unit or if the check is not
```

Comment:
"In order to check if it is safe to remove this unit, juju-verify needs to be explictly run against this subordinate unit, or the unit needs to be manually checked (if juju-verify does not support this charm yet)"